frontend/src/Components/EditableFields.tsx

```diff
@@ -33,31 +31,41 @@ export const EditableField = ({
     });
   };
 
-  const [groupBacking, setGroupBacking] = useState(group);
-
-  const ref = useRef<HTMLInputElement>(null);
-
+  // Given we now are not useing GroupSelect, there is a bit of code duplication
+  // here, but be careful if you want to refactor it! 'group' (unlike the other
+  // fields) is NOT stored in the workstation JSON!
   return (
     <Td>
       {fieldKey === "group" ? (
-        <GroupSelect onChange={handleSubmit}>
-          <AutoCompleteInput
-            ref={ref}
-            placeholder={"Unknown"}
-            // onSelect={() => {
-            //   setTimeout(() => {
-            //     ref.current?.focus();
-            //   }, 1);
-            //   setShouldFetch(false);
-            // }}
-            // onBlur={() => {
-            //   setShouldFetch(true);
-            // }}
-            onChange={(a) => setGroupBacking(a.target.value)}
-            value={groupBacking}
-          ></AutoCompleteInput>
-        </GroupSelect>
+        <Editable
+          defaultValue={group}
+          placeholder={placeholder}
+          textColor={pickCol(group)}
+          onSubmit={(a) => {
+            handleSubmit(a);
+          }}
+        >
+          <EditablePreview />
+          <EditableInput />
+        </Editable>
       ) : (
+        // <GroupSelect onChange={handleSubmit}>
```

Comment:
remove this pls

Reply:
https://github.com/gpuctl/gpuctl/pull/220/commits/f8760215e76926b4c14b1e26e8fea465acecbf83